repos.json

```diff
@@ -3,7 +3,7 @@
     "name": "attribution-reporting-api",
     "repo": "https://github.com/WICG/attribution-reporting-api",
     "spec": "https://wicg.github.io/attribution-reporting-api/",
-    "score": 3951.8999999999996
```

Comment:
repos.json is automatically generated on the server, so not sure this has an impact..